Added new volunteers

hrshkshri opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *all* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes #19554 
### This PR does the following
Added new volunteers and organization of existing volunteers into their respective sections

## Essential Checklist

- [x] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...", followed by a short, clear summary of the changes.
- [x] I have followed the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
- [x] I have assigned the correct reviewers to this PR (or will leave a comment with the phrase "@{{reviewer_username}} PTAL" if I don't have permissions to assign reviewers directly).
- [x] The linter/Karma presubmit checks pass on my local machine, and my PR follows the [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide)).
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)

## Proof that changes are correct


https://github.com/oppia/oppia/assets/108923011/4b12cde8-9bbc-4f63-9c7b-b074512b9d39


## PR Pointers

- Never force push! If you do, your PR will be closed.
- To reply to reviewers, follow these instructions: https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve
- Some e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR).
- See the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia's-code-owners-and-checks-to-be-carried-out-by-developers) for what code owners will expect.

hrshkshri committed:
added new volunteers

hrshkshri committed:
lint

hrshkshri committed:
Merge branch 'develop' into issue-19554

hrshkshri commented:
cc/ @seanlip

hrshkshri commented:
Hey @Lawful2002 cc/ @seanlip PTAL

hrshkshri commented:
Hi @seanlip, I noticed that in all three options ( share, feedback, report exploration )
Share is not translated into other languages but only into English.
Although this problem is unrelated to this particular issue this might be a serious problem.

seanlip commented:
> Hi @seanlip, I noticed that in all three options ( share, feedback, report exploration ) Share is not translated into other languages but only into English. Although this problem is unrelated to this particular issue this might be a serious problem.

@hrshkshri I'm not sure what the relation of your comment is to this issue, but if you've found a defect on the site, please feel free to file it as a new issue. Thanks.

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -114,20 +121,38 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'leveraging technology to enhance and enrich the learning ' +
+            'experience, making education accessible and engaging for all.”'
+          ),
+          name: '- liwei zhang',
```
Capitalize: Liwei Zhang

Also, the "- " should not be part of the name. It should be part of the HTML. Please drop it from the data here and only include it in the HTML code since it's more of a styling thing and not part of the actual data.

hrshkshri replied:
Done

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+          content: (
+            '“At Oppia, my love for writing stories as a ' +
+            'creative intertwines with my passion for ' +
+            'education. “'),
```
There is an extra space before the endquote, and I think the endquote is a different character than those used in other quotes?

Also, if the quotes are the same across all the blurbs then make that part of the HTML instead and omit them from the data.

hrshkshri replied:
Done

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'hat a decent education is for everyone and I feel grateful to ' +
+            ' be able to help thousands of people worldwide on this journey.“'),
+          name: '- Giovana Alonso',
+          type: 'Brazilian, Portuguese Translator'
```
Drop the comma. "Brazilian Portuguese" is one language.

hrshkshri replied:
Done

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
-          type: 'Translations/Voiceovers Coordinator'
+            '“I was always told that quality education is only for those who ' +
+            'are privileged. But by volunteering at Oppia, I prove every day ' +
+            'hat a decent education is for everyone and I feel grateful to ' +
```
hat --> that

hrshkshri replied:
done

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            '“I was always told that quality education is only for those who ' +
+            'are privileged. But by volunteering at Oppia, I prove every day ' +
+            'hat a decent education is for everyone and I feel grateful to ' +
+            ' be able to help thousands of people worldwide on this journey.“'),
```
Extra space at start.

hrshkshri replied:
done

seanlip reviewed (changes requested):
Hi @hrshkshri, thanks -- it looks pretty good! I left some inline notes.

For general comments, I only have two comments:

- Could you please crop the photos so that they are all the same size? Please ask the contributors for updated ones if needed. In your video, when the photos have different heights, it looks really messy.
- Did you get the email I sent you re Abha Barge? I don't see her included in the practice questions / lessons section, please add her.

Thanks!

hrshkshri committed:
added volunteer and fixed typo

hrshkshri committed:
Merge branch 'issue-19554' of https://github.com/hrshkshri/oppia into issue-19554

hrshkshri committed:
typo

seanlip reviewed (changes requested):
@hrshkshri The photo carousel looks a lot better now, thanks. Only one thing: could you please get better photos from Yiga and Erio that don't have the cropping on the sides? Maybe ask them to take a horizontal one.

Also, I don't think you've addressed the second point here: https://github.com/oppia/oppia/pull/19591#discussion_r1466604009 or the general point here: https://github.com/oppia/oppia/pull/19591#discussion_r1466606561 . I still see quotes and hyphens in your JS data. Please don't say that a comment is done when it's not done.

hrshkshri commented:
> @hrshkshri The photo carousel looks a lot better now, thanks. Only one thing: could you please get better photos from Yiga and Erio that don't have the cropping on the sides? Maybe ask them to take a horizontal one.

Sent a mail and cc'd you. 

> Also, I don't think you've addressed the second point here: [#19591 (comment)](https://github.com/oppia/oppia/pull/19591#discussion_r1466604009) or the general point here: [#19591 (comment)](https://github.com/oppia/oppia/pull/19591#discussion_r1466606561) . I still see quotes and hyphens in your JS data. Please don't say that a comment is done when it's not done.

Sorry, Forgot to push the changes : )

hrshkshri committed:
removed quote

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'way and it is broken down in a language they understand. Also, ' +
+            'no matter how good a product is, people will not know it ' +
+            'without effective marketing.'),
+          name: '- Yiga',
```
Yiga Ikpae

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'I believe education is the route to social upliftment ' +
+            'and progress. At Oppia, I get to be a part of this' +
+            'movement to provide free and accessible education for all.'),
+          name: '- Jay',
```
Jay Vivarekar

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'Oppia\'s mission and values inspire me daily. I volunteer ' +
+            'because everyone deserves an education and helping to market ' +
+            'this opportunity is a privilege'),
+          name: '- Jennifer',
```
Jennifer Nunez

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'education but also fosters a sense of community and inspiration,' +
+            'encouraging more individuals to engage with Oppia\'s ' +
+            'educational resources.'),
+          name: '- Erio',
```
Erio Crucecia

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'gift of education, you are giving happiness and smiles on ' +
+            'faces, and that is the biggest gift…and YES, as a volunteer I ' +
+            'am playing my part and helping kids all over the world.'),
+          name: '- Kanupriya Goel',
```
Isn't Kanupriya's surname Gupta?

seanlip commented on core/templates/pages/volunteer-page/volunteer-page.component.ts:
```diff
@@ -101,133 +106,230 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
+            'it\'s a passion. Contributing to Oppia affords to extend this ' +
+            'impact to the lives of many more children, and I am' +
+            'truly grateful for the privilege.'),
+          name: '- Pretty',
```
Pretty Agu

seanlip reviewed (changes requested):
@hrshkshri For the third time, the hyphens are still in the JS data. It is not great to have to remind about the same exact thing 3 times -- please read the reviewer comments carefully.

hrshkshri committed:
removed hyphens

hrshkshri committed:
Full name of volunteers

hrshkshri committed:
full name of volunteers

hrshkshri commented:
> @hrshkshri For the third time, the hyphens are still in the JS data. It is not great to have to remind about the same exact thing 3 times -- please read the reviewer comments carefully.

Very sorry about that, removed hyphens and added the full names of the volunteers, I hope I did not miss anything this time.
Sorry again.

seanlip reviewed (commented):
Thanks Harsh -- looks good. I'll approve once the photos are updated, but nothing else from my end!

hrshkshri committed:
New image of volunteer

Lawful2002 reviewed (approved):
Thanks @hrshkshri, LGTM from my side, but please look at the failing CI test.

hrshkshri committed:
new image of volunteer

hrshkshri commented:
> Thanks Harsh -- looks good. I'll approve once the photos are updated, but nothing else from my end!

Hey @seanlip PTAL

Photos are updated.
Here is the final look

https://github.com/oppia/oppia/assets/108923011/83cda33a-9632-4b70-8c2b-757e630dd64f

seanlip commented:
@hrshkshri It looks amazing. Thanks a lot! LGTM.

seanlip commented:
@hrshkshri This failed in the merge queue due to incomplete coverage of frontend tests. Could you PTAL? Thanks!

hrshkshri committed:
Merge branch 'develop' into issue-19554

hrshkshri committed:
Merge branch 'develop' into issue-19554

hrshkshri commented:
Hi @seanlip 
Sorry for the late response, 
The failed frontend test in the merge queue is related to `router.service.ts` which does not fail in CI checks (all of the frontend tests passed in CI checks) 
Also, this PR is not changing anything related to `router.service.ts`

hrshkshri commented:
> Hi @seanlip Sorry for the late response, The failed frontend test in the merge queue is related to `router.service.ts` which does not fail in CI checks (all of the frontend tests passed in CI checks) Also, this PR is not changing anything related to `router.service.ts`

PTAL

seanlip commented:
@hrshkshri I wasn't talking about a test failure, the issue was a coverage issue. See [this link](https://github.com/oppia/oppia/wiki/Frontend-tests#coverage-reports). If your PR doesn't pass the merge queue then we can't merge it, so I suggest trying to figure out the root cause of the error (you can view the logs of the run from the merge queue).

hrshkshri committed:
Merge branch 'develop' into issue-19554
